test/reducers/navigator.spec.js

```diff
@@ -56,4 +56,18 @@ describe('navigate reducer', () => {
     expect(state.stack.count()).to.be.equal(2);
     expect(state.stack.first()).to.be.equal('test');
   });
+
+  it('expect REPLACE_AT_INDEX to replace the route at the index passed', () => {
```

Comment:
add test for a negative scenario (when you trying to replace an index which is out of boundaries)